src/redux/Books/Books.js

```diff
@@ -1,27 +1,71 @@
+import axios from 'axios';
+
 const CREATE_BOOK = 'bookstore/books/create';
 const REMOVE_BOOK = 'bookstore/books/remove';
+const API_URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/IQMNjP1IzehTHPkD3U0K/books';
+const GET_BOOKS = 'bookstore/books/get';
 const initialState = {
   books: [],
 };
+
+// FUNCTIONS
+const postBook = (book) => {
+  axios.post(`${API_URL}`, book);
+};
+const deleteBook = (id) => {
+  axios.delete(`${API_URL}${id}`);
+};
```

Comment:
- [ ] The delete request is failing with a 404 error, please investigate the issue.

Reply:
Thanks @omar-labana